addons/agent_mirrorpeer_controller.go

```diff
@@ -310,59 +307,27 @@ func (r *MirrorPeerReconciler) labelRBDStorageClasses(ctx context.Context, stora
 }
 
 func (r *MirrorPeerReconciler) createS3(ctx context.Context, mirrorPeer multiclusterv1alpha1.MirrorPeer, scNamespace string) error {
-	noobaaOBC, err := r.getS3bucket(ctx, mirrorPeer, scNamespace)
-	if err != nil {
-		if errors.IsNotFound(err) {
-			r.Logger.Info("ODR ObjectBucketClaim not found, creating new one", "MirrorPeer", mirrorPeer.Name, "namespace", scNamespace)
-			err = r.SpokeClient.Create(ctx, noobaaOBC)
-			if err != nil {
-				r.Logger.Error("Failed to create ODR ObjectBucketClaim", "error", err, "MirrorPeer", mirrorPeer.Name, "namespace", scNamespace)
-				return err
-			}
+	logger := r.Logger.With("MirrorPeer", mirrorPeer.Name)
```

Comment:
This is already set by the main `Reconcile()` function.

Reply:
done